### DISCUSSION TITLE
Osu!Lazer doesn't let me switch desktops on Mac

### DISCUSSION THREAD

**Discussion by ayvacs** [ppy/osu#17207] [Q&A] [answered]
When I launch Osu!Lazer it locks the cursor to the app, doesn't allow me to switch to any other desktops or take the game out of fullscreen, and locks me from leaving the game other than pressing the exit button. Is there any way to disable that?

macOS 12.2.1
M1

**Comment by frenzibyte** [comment, marked as answer]
Running "Fullscreen" mode will make it run exclusively on macOS, disabling access to any external tools offered by the OS. You may want to switch to "Borderless" in that case.

I'm not sure myself if we still want to provide running exclusive fullscreen on macOS, as I recall it being some sort of a deprecated functionality. But I believe it will still give the benefit of reduced latency.

See https://github.com/ppy/osu/discussions/13351#discussioncomment-831131.
